stylesheets/components/modals/modal.css

```diff
@@ -5,30 +5,45 @@
     position: relative;
 }
 
+.modal-handler{
+    width: fit-content;
+    margin: auto;
+}
+
+.modal-bg-content{
+    display: none;
+    position: fixed;
+    z-index: 1;
```

Comment:
you can give z-index values and sizes vales in variable it will make easier for you to modify your code in the future